app/js/lib/foxbox/foxbox.js

```diff
@@ -260,6 +267,10 @@ export default class Foxbox extends Service {
     }
 
     this[p.nextPollTimeout] = setTimeout(() => {
+      if (!this.isLoggedIn) {
```

Comment:
__nit:__ maybe just add this new condition to the ```if``` branch above (```if (!this[p.isPollingEnabled] || this[p.nextPollTimeout] || !this.isLoggedIn) ```)? In this case we'll have it inside two main places at the very beginning (```schedulePoll``` and ```refreshServicesByPolling```)